sunpy/net/helioviewer.py

```diff
@@ -40,27 +40,19 @@ def get_data_sources(self, **kwargs):
 
         return self._get_json(params)
 
-    def get_closest_image(self, date, **kwargs):
+    def get_closest_image(self, date, sourceId, **kwargs):
         """Finds the closest image available for the specified source and date.
 
         For more information on what types of requests are available and the
         expected usage for the response, consult the Helioviewer API
-        documentation: http://legacy.helioviewer.org/api/docs/v1/ .
+        documentation: http://api.helioviewer.org/docs/v2/ .
 
         Parameters
         ----------
         date : `datetime.datetime`, `str`
             A string or datetime object for the desired date of the image
-        observatory : string
-            (Optional) Observatory name
-        instrument : string
-            (Optional) instrument name
-        detector : string
-            (Optional) detector name
-        measurement : string
-            (Optional) measurement name
         sourceId : int
-            (Optional) data source id
+            An integer which tells the data source id
```

Comment:
It is an integer? This part needs more explanation I think for users.

Reply:
Ok

Reply:
Having never used this before, how does someone get/know the sourceID?

Reply:
Sure I will make it more descriptive for the users